bug: Missing CommonJS compilation artifacts

I was trying to unit test my components after decomposition. But as soon as the test runs it fails with this message:

    Test suite failed to run
    Cannot find module './api/index.cjs' from 'node_modules/@assistant-ui/react/dist/index.js'

    Require stack:
      node_modules/@assistant-ui/react/dist/index.js
      src/components/Chatbot/components/ThreadWelcome.tsx
      src/components/Chatbot/Chatbot.tsx
      src/components/Chatbot/tests/Chatbot.test.tsx

I checked the main index.js file in the dist folder and it tries to import these .cjs files:

    // src/index.ts
    var src_exports = {};
    __export(src_exports, {
      INTERNAL: () => INTERNAL
    });
    module.exports = __toCommonJS(src_exports);
    __reExport(src_exports, require("./api/index.cjs"), module.exports);
    __reExport(src_exports, require("./context/index.cjs"), module.exports);
    __reExport(src_exports, require("./model-config/index.cjs"), module.exports);
    __reExport(src_exports, require("./primitive-hooks/index.cjs"), module.exports);
    __reExport(src_exports, require("./primitives/index.cjs"), module.exports);
    __reExport(src_exports, require("./runtimes/index.cjs"), module.exports);
    __reExport(src_exports, require("./types/index.cjs"), module.exports);
    __reExport(src_exports, require("./ui/index.cjs"), module.exports);
    var INTERNAL = __toESM(require("./internal.cjs"));
    // Annotate the CommonJS export names for ESM import in node:
    0 && (module.exports = {
      INTERNAL,
      ...require("./api/index.cjs"),
      ...require("./context/index.cjs"),
      ...require("./model-config/index.cjs"),
      ...require("./primitive-hooks/index.cjs"),
      ...require("./primitives/index.cjs"),
      ...require("./runtimes/index.cjs"),
      ...require("./types/index.cjs"),
      ...require("./ui/index.cjs")
    });

But no .cjs file exists in the repo, they are all .js instead.

Thank you!

--- a/.changeset/chilled-boats-thank.md
+++ b/.changeset/chilled-boats-thank.md
@@ -1,0 +1,7 @@
+---
+"@assistant-ui/react-markdown": patch
+"@assistant-ui/react-trieve": patch
+"@assistant-ui/react": patch
+---
+
+fix: cjs builds

--- a/packages/tsbuildutils/src/index.ts
+++ b/packages/tsbuildutils/src/index.ts
@@ -45,7 +45,11 @@ const transpileTypescript = async () => {
     splitting: false,
     silent: true,
 
-    esbuildPlugins: [esbuildPluginFilePathExtensions()],
+    esbuildPlugins: [
+      esbuildPluginFilePathExtensions({
+        cjsExtension: "js",
+      }),
+    ],
   });
 };
 

